src/systems/logical_audio_sensor_plugin/LogicalAudio.hh

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_LOGICAL_AUDIO_SENSOR_PLUGIN_LOGICALAUDIO_HH_
+#define IGNITION_GAZEBO_SYSTEMS_LOGICAL_AUDIO_SENSOR_PLUGIN_LOGICALAUDIO_HH_
+
+#include <string>
+
+#include <ignition/gazebo/components/LogicalAudio.hh>
+#include <ignition/gazebo/config.hh>
+#include <ignition/math/Pose3.hh>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace logical_audio
```

Comment:
Is the `logical_audio` namespace okay here? Or should it be a sub-namspace of `systems`?

Reply:
Sounds good, I'll leave the `logical_audio` namespace as is for now. If someone else points out a reason to change it, then I can update it accordingly.